src/main/java/com/celements/search/web/classes/WebSearchClassPackage.java

```diff
@@ -12,7 +12,7 @@
 @Component(WebSearchClassPackage.NAME)
 public class WebSearchClassPackage extends AbstractClassPackage {
 
-  public static final String NAME = "webSearch";
+  public static final String NAME = "websearch";
```

Comment:
package is activated with [this PR](https://github.com/celements/celements-web/pull/52/files#diff-91d27ecf923bd14b7d4cb99a7e9ec6b5)